src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/types/ContactType.kt

```diff
@@ -4,4 +4,5 @@ enum class ContactType {
   HOME,
   MOBILE,
   EMAIL,
+  ALTERNATE,
```

Comment:
where has this come from?

Reply:
Comes from the 'for contact types we need to add “alternate_telephone_number”' But i guess this is just another version of 'HOME', 'MOBILE' etc.

Reply:
Do we care?
Telephone numbers and emails is all there actually are. Home/mobile/whatever has no meaning for us I don't think 
I wonder if we should just store them like we do middle names?
Just thinking out loud